src/pages/workspace/accounting/xero/XeroOrganizationConfigurationPage.tsx

```diff
@@ -42,6 +47,7 @@ function XeroOrganizationConfigurationPage({
         }
 
         updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.XERO, 'tenantID', keyForList);
+        Navigation.goBack(ROUTES.WORKSPACE_ACCOUNTING.getRoute(policyID));
```

Comment:
Note: This caused an issue that pushed two FullScreenNavigator to the route stacks.

Fixed in  - https://github.com/Expensify/App/issues/42924